report/report.tex

```diff
@@ -97,29 +97,24 @@
 %----------------------------------------------------------------------------------------
 
 \section{Introduction}
+In the globalized economy of today, consumers are presented with a range of products to choose from when shopping. This can present some difficulty, as quality of the product is not always clear from its presentation. The customer review seeks to solve this by providing feedback on the purchase. Still, determining the value of a review can be difficult, as reviews can be misleading or provide little information.
 
-\lettrine[nindent=0em,lines=3]{L} orem ipsum dolor sit amet, consectetur adipiscing elit.
-\blindtext % Dummy text
-
-\blindtext % Dummy text
+Voting systems set in place to identify helpful reviews help consumers circumvent this issue, yet these reviews still get lost in the sheer number of unlabeled reviews. Our machine learning model aims to further filter reviews by analyzing verbosity, sentiment and other aspects, to discover what makes a review helpful.
 
 %------------------------------------------------
 
 \section{Methods}
+We have made use of the agile and iterative research approach proposed in Nunamaker et al., to continuously gain feedback on the activities involved in our process.
 
-Maecenas sed ultricies felis. Sed imperdiet dictum arcu a egestas. 
-\begin{itemize}
-\item Donec dolor arcu, rutrum id molestie in, viverra sed diam
-\item Curabitur feugiat
-\item turpis sed auctor facilisis
-\item arcu eros accumsan lorem, at posuere mi diam sit amet tortor
-\item Fusce fermentum, mi sit amet euismod rutrum
-\item sem lorem molestie diam, iaculis aliquet sapien tortor non nisi
-\item Pellentesque bibendum pretium aliquet
-\end{itemize}
-\blindtext % Dummy text
+Our research aims at creating a predictive model for identifying helpful Amazon reviews. Therefore, the data used in this study is a subset of Amazon product reviews divided into categories, with each product and reviewer having at least 5 reviews.
```

Comment:
Is the 5-core used anymore?

Reply:
http://deepyeti.ucsd.edu/jianmo/amazon/index.html
Mener at vi bare har downloadet reviews filerne ikke 5-core, ihvertfald til at køre modellerne

Reply:
Nå, okay. Men det finder vi ud af i en anden section :D

Reply:
Alrighty :D